erpnext_documentation/www/docs/user/manual/en/CRM/newsletter.md

```diff
@@ -12,6 +12,61 @@ You need to create [Email Groups](/docs/user/manual/en/CRM/email_group) before s
 To access the Newsletter list, go to:
```

Comment:
@Mangesh-Khairnar, this file's contents are duplicated, update according to what you want to keep

Reply:
added changes